source/core/src/main/com/csse3200/game/components/combat/CombatEnvironmentDisplay.java

```diff
@@ -20,9 +29,18 @@ public class CombatEnvironmentDisplay extends UIComponent {
   private static final Logger logger = LoggerFactory.getLogger(CombatEnvironmentDisplay.class);
   private static final float Z_INDEX = 2f;
   private Table table;
+  private CombatTerrainFactory combatTerrainFactory;
```

Comment:
Actually I don't think CombatEnvironmentDisplay is being used anymore is it? We should be able to delete it from main